tests/e2e/cypress/cypress/integration/templates/templates-test.spec.js

```diff
@@ -27,7 +27,7 @@ before(function () {
   });
   cy.fixture('templates').then(function (templateData) {
     this.templates = templateData;
-    this.templateName = this.templates.templateA.name + cy.getCurrentDateAndHour();
+    this.templateName = this.templates.templateA.name + ' ' + cy.getCurrentDateAndHour();
```

Comment:
`[this.templates.templateA.name, cy.getCurrentDateAndHour()].join(' ')`

Reply:
Done